src/pages/index.tsx

```diff
@@ -377,29 +376,29 @@ const HomeHero: FunctionComponent<HomeHeroProps> = ({ onOpenModal }) => {
     const isMobile = windowWidth < breakpoints.sm
 
     return (
-        <ContentSection className="flex items-center justify-center" parentClassName="!py-0">
+        <ContentSection className="flex items-center justify-center" parentClassName="!py-0 !pb-16 !bg-gray-50">
             <div className="mx-auto flex flex-col items-center justify-center text-center">
                 <div className="mx-auto flex flex-col items-center pb-16 pt-8 md:w-[680px] md:pb-[70px] md:pt-16">
-                    <h1 className="mb-8 w-full text-center text-white md:mb-6">
+                    <h1 className="mb-8 w-full text-center text-gray-700 md:mb-6">
                         Understand and write code blazingly fast
                     </h1>
-                    <p className="mb-10 text-2xl font-normal leading-[30px] -tracking-[0.25px] text-[#FFFFFF99] md:mb-8">
+                    <p className="mb-10 text-2xl font-normal leading-[30px] -tracking-[0.25px] text-gray-500 md:mb-8">
                         Sourcegraph allows developers to rapidly search, write, and understand code by bringing insights
                         from their entire codebase right into the editor
                     </p>
                     <button
                         type="button"
-                        className={classNames('btn btn-primary-dark')}
+                        className="btn btn-primary"
                         title="free cody"
                         onClick={() => onOpenModal('top')}
                     >
                         Get Cody for free
                     </button>
                 </div>
                 {!isMobile && (
-                    <div className="w-full overflow-hidden lg:w-[1062px]">
+                    <div className="w-full lg:w-[1062px]">
                         <video
-                            className="relative bottom-[-6px] rounded-t-lg"
+                            className="rounded-lg shadow-md"
```

Comment:
1. Looks like the video is missing its gray-200 border
2. There is a weird black line on the bottom of the video. Is that something we can crop out via styling? Or will we have to re-upload a cropped file?
<img width="1294" alt="Screenshot 2024-07-08 at 3 58 30 PM" src="https://github.com/sourcegraph/about/assets/102541294/2fd9ea55-832d-4590-b15f-c858427f05d2">

Reply:
Hi @traceyljohnson 
Let us re-upload a cropped file.
Thank you

Reply:
Hmm. If its not a simple fix, lets go ahead and leave the video as is, but update the hero format to something like this: https://www.figma.com/design/fudljjrGrUg5cDYTAo4Hra/Homepage?node-id=1634-4411&focus-id=3851-43000&m=dev

Reply:
@gitstart-sourcegraph I'm noticing there is an extra 24px of padding on the left and right of the navbar that shouldn't be there. Can you remove? 
<img width="362" alt="Screenshot 2024-07-12 at 8 39 19 AM" src="https://github.com/user-attachments/assets/ccc6e677-39dc-475b-bd29-709a0b85b818">